### PR TITLE
add ngx-masonry [WIP]

### PR DESCRIPTION
**Pull request by x0rzkov**
https://github.com/wynfred/ngx-masonry/

## Usage

Import `NgxMasonryModule` into your app's modules:

```typescript
import { NgxMasonryModule } from 'ngx-masonry';

@NgModule({
  imports: [
    NgxMasonryModule
  ]
})
```

```typescript
@Component({
  selector: 'my-component',
  template: `
     <ngx-masonry>
       <div ngxMasonryItem class="masonry-item" *ngFor="let item of masonryItems">
        {{item.title}}
      </div>
     </ngx-masonry>
     `,
  styles: [
    `
       .masonry-item { width: 200px; }
     `
  ]
})
class MyComponent {
  masonryItems = [
    { title: 'item 1' },
    { title: 'item 2' },
    { title: 'item 3' },
    { title: 'item 4' },
    { title: 'item 5' },
    { title: 'item 6' }
  ];
}
```

### CODE REVIEW

**Comment by x0rzkov** [comment]
Any thoughts about this one, my dear Viktor ?

File: rf-ng/src/app/article-list/article-list.css [hunk 1]
```diff
@@ -64,3 +64,7 @@ list-item {
         transform: translate(500%);
     }
 }
+
+.masonry-item { 
+    width: 250px;
```
Comment (urandom) [inline rf-ng/src/app/article-list/article-list.css]: There seem to be trailing whitespaces here that need to be removed

File: rf-ng/src/app/article-list/article-list.component.ts [hunk 2]
```diff
@@ -5,9 +5,7 @@ import { IPageInfo } from 'ngx-virtual-scroller';
 import { Observable, Subscription ,  BehaviorSubject, interval } from "rxjs";
 import * as moment from 'moment';
 import { scan, map, switchMap, startWith } from "rxjs/operators";
-
-
-
+import { NgxMasonryModule } from 'ngx-masonry';
```
Comment (urandom) [inline rf-ng/src/app/article-list/article-list.component.ts]: you have to import this in the app.module and add it to the list of module imports.

**Comment by urandom** [comment]
The masonry layout really doesnt play well with virtual scrolling, which i consider more important. To bridge the gap a little, i made the cards have the same height

**Pull request closed without merging**